crates/flox/src/commands/environment.rs

```diff
@@ -203,9 +203,8 @@ impl EnvironmentCommands {
                     .catalog(&flox.nix(Default::default()), &flox.system)
                     .await
                     .context("Could not get catalog")?;
-                // let installed_store_paths = env.installed_store_paths(&flox).await?;
 
-                println!("Packages in {}:", env.environment_ref());
+                println!("Packages in {}:", env.name());
```

Comment:
I'd prefer keeping environment refs as the consistent way of parsing and printing environment names but as names are currently just that, names, no need to think too hard here now.

Reply:
From an API perspective `name` is much clearer than `environment_ref`. I'd be fine calling `environment_ref` under the hood.

Reply:
This was actually already using `environment_ref.name` under the hood

Reply:
Well i just want to avoid us adding basically

```
if let Some(owner) = env.owner() {
	print!("{owner}/");
}
println!("{}, env.name());
```

whenever we need to print the name/identifier for an environment, which is exactly what environment_ref is there for.

We can rename that struct/method if its not descriptive enough though (or do you mean for `name()` to return a formatted string instead?)
Until we have remote/managed envs that shouldn't make much of a difference..